toast.css

```diff
@@ -0,0 +1,59 @@
+.toast {
+  background-color: azure;
+  min-width: 150px;
+  height: 30px;
+  font-family: "Roboto";
+  font-size: 20px;
+  padding: 10px;
+
+}
+
+.toastcloseicon {
+  border: none;
+  float: right;
+  padding-left: 7px;
+  color: black;
+}
+
+.toast-success {
+  background-color: #78D878;
+}
+
+.toast-error {
+  background-color: #ED2939;
+}
+
+.toast-inform {
+  background-color: #00FFFF;
+}
+
+#toast-success-demo {
+  visibility: hidden;
+  min-width: 250px;
+  margin-left: -125px;
+  background-color: #78D878;
+  color: black;
+  text-align: center;
+  border-radius: 2px;
+  padding: 16px;
+  position: fixed;
+  z-index: 1;
+  left: 50%;
+  top: 30px;
+  font-size: 17px;
+
+}

```

Comment:
Same changes here as well, use rem units and colour variables.
Remove the blank space at line 44.